doc/UserGuide/BuildingRunningTesting/RunningUPP.rst

```diff
@@ -6,122 +6,114 @@
+It is recommended that the user refer to the ``run_upp`` script while reading this overview. All user-modified variables are contained at the top of the ``run_upp`` script in the user-edit section, along with a brief description. Descriptions below follow the ``run_upp`` script.
 
-1. Set up basic path variables:
+#. Set up basic path variables:
 
    * ``TOP_DIR``: Top level directory for building and running UPP
-   * ``DOMAINPATH``: Working directory for this run
+   * ``POSTPRD_DIR``: Working directory for this run
    * ``UPP_HOME``: Location of the **UPP** directory
    * ``POSTEXEC``: Location of the **UPP** executable
-   * ``modelDataPath``: Location of the model output data files to be processed by the UPP
-   * ``txtCntrlFile``: Name and location of the flat text file that lists desired fields for output.
+   * ``modelDataPath``: Location of the model output files to be processed by the UPP
+   * ``txtCntrlFile``: Full path to the flat text file that lists desired fields for output. 
+      
+      * :term:`GFS` text files are located in ``UPP/parm/gfs`` and include ``postxconfig-NT-gfs-f00-two.txt`` (0-hour lead time) and ``postxconfig-NT-gfs-two.txt`` (all other lead times).
+      * :term:`LAM` text files are located in ``UPP/parm/rrfs`` and include ``postxconfig-NT-rrfs.txt``. 
+
+   * ``CRTMDIR``: Path to simulated synthetic satellite files (required for the LAM implementation)
 
    .. note::
-      For FV3, the scripts are configured such that UPP expects the flat text file to be in ``/parm``,
-      and the postprocessor working directory to be called ``/postprd``, all under ``DOMAINPATH``.
+      For FV3, the scripts are configured such that UPP expects the flat text file to be in ``parm``,
+      and the postprocessor working directory to be called ``postprd``.
       This setup is for user convenience to have a script ready to run; paths may be modified, but be
       sure to check the run script to make sure settings are correct.
 
-2. Specify dynamical core being run:
+#. Specify model configuration being run in the ``model`` field. Valid options: Global Forecast System (``GFS``) or Limited Area Model (``LAM``). 
 
-   * ``model``: Which model is used? ("GFS" or "LAM" - Limited Area Model)
+   .. note::
+      Forecast output should include ``atmf*``, ``sfcf*``, and ``GFSPRS*`` files when running with ``model=GFS``, and it should include ``phyf*``, ``dynf*``, ``NATLEV*``, and ``PRSLEV*`` files when ``model=LAM``. 
 
-3. Specify the format for the input model files and output UPP files:
+#. Specify the format for the input model files and output UPP files:
 
    * ``inFormat``: Format of the model data ("netcdfpara")
    * ``outFormat``: Format of output from UPP ("grib2")
 
-4. Specify the forecast cycles to be post-processed:
+   .. COMMENT: Maybe we shouldn't list this, since it's set, and the user can't/shouldn't change it? ^
```

Comment:
@jdkublnick What do you think about this? Maybe I should move it out of the user-edit section in the script? I think it's a relic from when there was also nemsio format?